File: Server-MovieTicketPurchaseApp/models/Seat.js
const mongoose = require('mongoose')
const Schema = mongoose.Schema

const Seat = new Schema({
    cinemaHallId: {type: String},
    seatName: {type: String},
    status: {type: Number} // 0 ghế trông, 1 đang chọn, 2 đã đặt
})
module.exports = mongoose.model("seat",Seat)

File: MovieTicketPurchaseApp/app/src/main/java/com/lhb/movieticketpurchaseapp/model/Seat.kt
package com.lhb.movieticketpurchaseapp.model

//data class Seat(
//    val id: String,
//    val cinemaHallId: String,
//    val seatName: String,
//    val status: Int // 0 is empty, 1 is selected, 2 is ordered
//)
data class Seat(
    val id: String,
    var row: Char,
    val number: Int,
    var status: SeatStatus
)